TFNativeOp.py

```diff
@@ -425,6 +436,8 @@ def _make_mod(self):
         from glob import glob
         for f in glob("%s/.libs/*.so" % numpy_dir):
           f = os.path.basename(f)
+          if self.blas_lib is not None and self.blas_lib not in f:
+            continue
           if f.startswith("lib"):
             f = f[3:]
           if f.endswith(".so"):
```

Comment:
The `assert` I was mentioning should be added at some point in the end.
Maybe actually that filtering should be done also there, so you don't need to have it before.